docs/learn/learn-runtime-upgrades.md

```diff
@@ -10,35 +10,24 @@ slug: ../learn-runtime-upgrades
 
 Runtime upgrades allow Polkadot to change the logic of the chain, without the need for a hard fork.
 
-## Forking vs Forkless
+## Forkless Upgrades
 
-When we talk about "forking", we are referring to the act of invoking the `fork` system call [within
-a system], where a process creates a copy of itself. In software development, this is the act of
-forking source code from one software package that creates a separate piece of that software.
+You may have come across the term "hard fork" before in the blockchain space. A **hard fork** occurs
+when a blockchain's logic changes such that nodes that do not include the new changes will not be
+able to remain in consensus with nodes that do. Such changes are backwards incompatible. Hard forks
+can be political due to the nature of the upgrades, as well as logistically onerous due to the
+number (potentially thousands) of nodes in the network that need to upgrade their software.
 
-You may have come across the terms "hard fork" and "soft fork" before in the blockchain space.
+Rather than encode the runtime (a chain's "business logic") in the nodes, Polkadot nodes contain a
+WebAssembly [execution host](learn-polkadot-host). They maintain consensus on a very low level and
+well established instruction set. The Polkadot runtime is stored on the Polkadot blockchain itself.
 
-A **hard fork** sees the blockchain's underlying technology (i.e. the protocol) radically changed
-and exists as a _different_ or _new_ version. This version is meant to be incompatible with the
-original version. This means that both blockchains exist simultaneously. Henceforth, a hard fork
-requires node operators to manually upgrade their nodes to the latest runtime version. In a
-distributed system, this is a complex process to coordinate and communicate.
+As such, Polkadot can upgrade its runtime by upgrading the logic stored on-chain, and removes the
```

Comment:
"upgrading the logic stored on-chain, and removes the" -> "upgrading the logic stored on-chain and remove the"

Reply:
No, it's the conjunction of two independent clauses. "comma and" is the correct way to handle it.